SlotsTheSpire/Assets/Scripts/GameManagers/BattleSystem.cs

```diff
@@ -7,10 +7,9 @@ public enum BattleState { START, PLAYERTURN, ENDTURN, ENEMYTURN, WON, LOSS }
 public class BattleSystem : MonoBehaviour
 {
 
-    public GameObject PlayerPrefab;
-    public GameObject enemyPrefab;
-    public FloatVariable turn, damage, playerShield, enemyHealth, enemyMaxHealth, playerHealth, floorLevel;
-    public BoolVariable isShielded;
+    public GameObject P_Prefab;
+    public GameObject E_Prefab;
+    public FloatVariable turn, enemyHealth, playerShield, enemyShield, enemyMaxHealth, playerHealth, floorLevel, E_outgoingDamage, P_outgoingDamage, P_incomingShield, E_incomingShield;
```

Comment:
Use naming convention (E_ for enemy) (P_ for player) for all variables including 'enemy' and 'player'